src/utils/translate/translateService.ts

```diff
@@ -64,8 +67,15 @@ export default function Translate(
   substitution: Record<string, string | number> | null = null,
   addSpaces: [boolean, boolean] | null = null
 ): string {
-  const translationDictionary: Json =
-    language === LanguageType.english ? (English as Json) : (French as Json);
+  let translationDictionary: Json = {}
+
+  if (language === LanguageType.english) {
```

Comment:
Wouldn't it be easier to just make this a lookup object as discussed, something like:

```
const translations = {
  "en": English,
  "fr": French,
  "de": German
}

const translationDictionary: Json = translations[language]
```